http/http.go

```diff
@@ -22,17 +24,26 @@ func Setup(etcdHost, discHost string) {
 	r.HandleFunc("/robots.txt", handlers.RobotsHandler)
 
 	// Only allow exact tokens with GETs and PUTs
-	r.HandleFunc("/{token:[a-f0-9]{32}}", handlers.TokenHandler).
-		Methods("GET", "PUT")
-	r.HandleFunc("/{token:[a-f0-9]{32}}/", handlers.TokenHandler).
-		Methods("GET", "PUT")
-	r.HandleFunc("/{token:[a-f0-9]{32}}/{machine}", handlers.TokenHandler).
-		Methods("GET", "PUT", "DELETE")
-	r.HandleFunc("/{token:[a-f0-9]{32}}/_config/size", handlers.TokenHandler).
-		Methods("GET")
+	r.HandleFunc("/{token:[a-f0-9]{32}}", handlers.TokenHandler).Methods("GET", "PUT")
```

Comment:
why change the whitespace? it shows an unnecessary diff so it's better to follow the existing pattern.